guides/common/modules/proc_configuring-the-active-directory-authentication-source-on-projectserver.adoc

```diff
@@ -120,10 +125,12 @@ $ kinit _ad_user_@_AD.EXAMPLE.COM_
 +
 [options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
-$ curl -k -u : --negotiate https://{foreman-example-com}/users/extlogin
+$ curl -k -u : --negotiate https://_{foreman-example-com}_/users/extlogin
 
-<html><body>You are being <a href="{foreman-example-com}/hosts">redirected</a>.</body></html>
+<html><body>You are being <a href="_{foreman-example-com}_/hosts">redirected</a>.</body></html>
 ----
++
+When external authentication is configured correctly, the `curl` command redirects you to `{foreman-example-com}/hosts`.
```

Comment:
```suggestion
When external authentication is configured correctly, the `curl` command redirects you to `\https://{foreman-example-com}/hosts`.
```

The backslash prevents asciidoctor to render a link to the "example.com" domain which does not exist. Similar to:

````
guides/common/modules/proc_creating-custom-alternate-content-sources-by-using-web-ui.adoc
13:For example, if your base URL is `\https://{server-example-com}` and your subpaths are `rhel10/` and `rhel9/`, then {Project} will search `\https://{server-example-com}/rhel10/` an [... 1 more match]
````